recipes/gstlal-inspiral/meta.yaml

```diff
@@ -0,0 +1,98 @@
+{% set name = "gstlal-inspiral" %}
+{% set version = "1.6.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz"
+  sha256: 66ef3f3f1552894cd6b1add4e49477e6a37c8adb23e5e1364fb5f001bfc57397
+  patches:
+    - missing-links.patch
+
+build:
+  ignore_run_exports:
+    # configure asks for it, but it is never actually used
+    - lalmetaio
+  number: 0
+  skip: true  # [win or py>=30]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make  # [unix]
+    - pkg-config >=0.18.0  # [unix]
+  host:
+    - glib
+    - gobject-introspection >=1.30.0
+    - gsl
+    - gstreamer
+    - gst-plugins-base
+    - gstlal >=1.5.0
+    - lal >=6.19.0
+    - lalinspiral >=1.8.0
+    - lalmetaio >=1.4.0
+    - numpy 1.14
+    - python
+  run:
+    - glib
+    - gobject-introspection
+    - gsl
+    - gstreamer
+    - gst-plugins-base
+    - gstlal >=1.5.0
+    - gstlal-ugly >=1.6.5
+    - h5py
+    - lal >=6.19.2
```

Comment:
I'm assuming not of these are being used as shared libraries? If they are you should use `pin_compatible` in some way to declare the ABI constraints.

Reply:
The `lal` libraries (at least) use `run_exports` to pin the ABI contraints, is that good enough?

Reply:
In that case I think you don't need the packages in `run` as the correct constraint will be put there.

Am I right in thinking there are now a lot of packages depending on `lal` and others? If so, you might want to add pins in https://github.com/conda-forge/conda-forge-pinning-feedstock so you can do migrations easily and keep a consistent definition of what is used for building. If you do this be careful that `>=6.19.2` prevents pins from being applied and should be removed.

Reply:
There are a lot of packages, but `lal` and friends are fast-moving and not particularly stable, so there would end up being a lot of migrations. I don't think this is worth it at the moment, but might be in the future, I'll keep it in mind.